src/main/java/com/disi/TravelPoints/model/Role.java

```diff
@@ -15,9 +16,17 @@ public class Role {
     @Enumerated(EnumType.STRING)
     @Column(name = "name")
     private RoleName name;
+    @OneToMany(mappedBy="role", cascade = {CascadeType.DETACH, CascadeType.MERGE, CascadeType.REFRESH})
+    private Set<User> users;
 
     public Role() {}
 
+    public Role(Long id, RoleName name, Set<User> users) {
+        this.id = id;
```

Comment:
Nu cred ca e nevoie de id in constructor, in mod normal ar trebui sa se asigneze unul in mod automat